Sources/Models/SubscribeData.swift

```diff
@@ -3,24 +3,28 @@ import Stripe
 
 public struct SubscribeData: Equatable {
   public var coupon: Stripe.Coupon.Id?
+  public var isOwnerTakingSeat: Bool
```

Comment:
this is to represent this state in the data that is submitted with the subscription form. we can use this to determine if we assign the subscription to the owner in the DB or not.